tests/Aspire.Hosting.Testing.Tests/TestingBuilderTests.cs

```diff
@@ -78,6 +80,41 @@ public async Task GetHttpClientBeforeStart(bool genericEntryPoint)
         Assert.Throws<InvalidOperationException>(() => app.CreateHttpClient("mywebapp1"));
     }
 
+    [LocalOnlyTheory]
+    [InlineData(false)]
+    [InlineData(true)]
+    public async Task SetsCorrectContentRoot(bool genericEntryPoint)
+    {
+        var appHost = await (genericEntryPoint
+            ? DistributedApplicationTestingBuilder.CreateAsync<Projects.TestingAppHost1_AppHost>()
+            : DistributedApplicationTestingBuilder.CreateAsync(typeof(Projects.TestingAppHost1_AppHost)));
+        await using var app = await appHost.BuildAsync();
+        await app.StartAsync();
+        var hostEnvironment = app.Services.GetRequiredService<IHostEnvironment>();
+        Assert.Contains("TestingAppHost1", hostEnvironment.ContentRootPath);
+    }
+
+    [LocalOnlyTheory]
+    [InlineData(false)]
+    [InlineData(true)]
+    public async Task SelectsFirstLaunchProfile(bool genericEntryPoint)
```

Comment:
Should we be adding an API for selecting the launch profile when using the test builder?